system/jlib/jtrace.cpp

```diff
@@ -594,66 +594,69 @@ class CSpan : public CInterfaceOf<ISpan>
     }
 
     /**
-     * Retrieves the Span's context as key/value pairs into the provided IProperties.
-     * Optionally, output follows OpenTelemetry Span context format for propogation 
+     * Retrieves the Span's client headers traceparent and tracestate
+     * Output follows OpenTelemetry Span context format for propogation 
      * accross process boundaries.
      *
-     * @param ctxProps IProperties container for span context key/value pairs.
-     * @param otelFormatted If true, output follows OpenTelemetry Span context format.
-     * @return True if the span context was successfully retrieved, false otherwise.
+     * @param clientHeaders IProperties container for client headers.
      */
-    bool getSpanContext(IProperties * ctxProps, bool otelFormatted) const override
+    void getClientHeaders(IProperties * clientHeaders) const
     {
-        if (ctxProps == nullptr)
-            return false;
+        if (clientHeaders == nullptr)
```

Comment:
minor: I don't think it is ever useful to call this if clientHeaders is null, so better to assert (e.g. assertex(clientHeaders)).  Otherwise the derived functions also need to check that clientHeaders also non-null for consistency.

same goes for getSpanContext()